### PR TITLE
Chore: Remove x from Org User

### PR DESCRIPTION
**Pull request by yangkb09**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref #41482

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/sqlstore/org_users.go [hunk 1]
```diff
@@ -98,147 +98,151 @@ func (ss *SQLStore) UpdateOrgUser(ctx context.Context, cmd *models.UpdateOrgUser
...
-	}
 
-	if query.Query != "" {
-		queryWithWildcards := "%" + query.Query + "%"
-		whereConditions = append(whereConditions, "(email "+dialect.LikeStr()+" ? OR name "+dialect.LikeStr()+" ? OR login "+dialect.LikeStr()+" ?)")
-		whereParams = append(whereParams, queryWithWildcards, queryWithWildcards, queryWithWildcards)
-	}
+		if query.Query != "" {
+			queryWithWildcards := "%" + query.Query + "%"
+			whereConditions = append(whereConditions, "(email "+dialect.LikeStr()+" ? OR name "+dialect.LikeStr()+" ? OR login "+dialect.LikeStr()+" ?)")
```
Comment (ying-jeanne) [inline pkg/services/sqlstore/org_users.go:132]:
```suggestion
			whereConditions = append(whereConditions, "(email "+ss.Dialect.LikeStr()+" ? OR name "+ss.Dialect.LikeStr()+" ? OR login "+ss.Dialect.LikeStr()+" ?)")
```

File: pkg/services/sqlstore/org_users.go [hunk 2]
```diff
@@ -98,147 +98,151 @@ func (ss *SQLStore) UpdateOrgUser(ctx context.Context, cmd *models.UpdateOrgUser
...
-	}
+		if query.UserID != 0 {
+			whereConditions = append(whereConditions, "org_user.user_id = ?")
+			whereParams = append(whereParams, query.UserID)
+		}
 
-	whereConditions = append(whereConditions, fmt.Sprintf("%s.is_service_account = ?", dialect.Quote("user")))
-	whereParams = append(whereParams, dialect.BooleanStr(false))
+		whereConditions = append(whereConditions, fmt.Sprintf("%s.is_service_account = ?", dialect.Quote("user")))
+		whereParams = append(whereParams, dialect.BooleanStr(false))
```
Comment (ying-jeanne) [inline pkg/services/sqlstore/org_users.go:119]:
```suggestion
		whereConditions = append(whereConditions, fmt.Sprintf("%s.is_service_account = ?", ss.Dialect.Quote("user")))
		whereParams = append(whereParams, ss.Dialect.BooleanStr(false))
```

File: pkg/services/sqlstore/org_users.go [hunk 3]
```diff
@@ -98,147 +98,151 @@ func (ss *SQLStore) UpdateOrgUser(ctx context.Context, cmd *models.UpdateOrgUser
...
-	}
 
-	if query.Query != "" {
-		queryWithWildcards := "%" + query.Query + "%"
-		whereConditions = append(whereConditions, "(email "+dialect.LikeStr()+" ? OR name "+dialect.LikeStr()+" ? OR login "+dialect.LikeStr()+" ?)")
-		whereParams = append(whereParams, queryWithWildcards, queryWithWildcards, queryWithWildcards)
-	}
+		if query.Query != "" {
+			queryWithWildcards := "%" + query.Query + "%"
+			whereConditions = append(whereConditions, "(email "+dialect.LikeStr()+" ? OR name "+dialect.LikeStr()+" ? OR login "+dialect.LikeStr()+" ?)")
```
Comment (ying-jeanne) [inline pkg/services/sqlstore/org_users.go:197]:
```suggestion
			whereConditions = append(whereConditions, "(email "+ss.Dialect.LikeStr()+" ? OR name "+ss.Dialect.LikeStr()+" ? OR login "+ss.Dialect.LikeStr()+" ?)")
```

**Comment by ying-jeanne** [comment]
It is a nice pr, just some forgotten dialect. to be replaced by the one of sqlstore structure.

**Comment by ying-jeanne** [comment]
you might need a rebase on main, the auto merged failed on drone

**Review by ying-jeanne — APPROVED** [review]
LGTM

**Review by idafurjes — APPROVED** [review]

**Pull request merged**